src/compile.rs

```diff
@@ -157,6 +161,59 @@ pub fn get_built_in_types_map() -> TypeMap {
     built_in_types
 }
 
+pub fn get_global_bindings(
+    p: &PolicyFile,
+    types: &mut TypeMap,
+    classlist: &mut ClassList,
+    file: &SimpleFile<String, String>,
+) -> Result<(), HLLErrors> {
+    for e in &p.policy.exprs {
+        if let Expression::Stmt(Statement::LetBinding(l)) = e {
+            let let_rvalue = ArgForValidation::from(&l.value);
+            let (variant, bound_type) = match let_rvalue {
+                ArgForValidation::List(v) => {
+                    let ti_vec = argument_to_typeinfo_vec(&v, types, classlist, None, file)?;
+                    let variant = type_slice_to_variant(&ti_vec, types)?;
+                    (
+                        variant.name.as_ref(),
+                        BoundTypeInfo::List(v.iter().map(|s| s.to_string()).collect()),
+                    )
+                }
+                a => {
+                    let ti = argument_to_typeinfo(&a, types, classlist, None, file)?;
+                    if ti.name.as_ref() == "perm" {
```

Comment:
I actually kind of prefer something more global here, since "perm" has special meaning in the code but not the Cascade grammar.  I think I'd prefer to do that as a follow-on PR though since it could probably use some independent discussion on it's own.